src/Backoffice/Task/Domain/Models/Task.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Task\Domain\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Lightit\Backoffice\Employee\Domain\Models\Employee;
+
+/**
+ * @property int                          $id
+ * @property string                       $title
+ * @property string                       $description
+ * @property string                       $status
+ * @property int|null                     $employee_id
+ * @property \Carbon\CarbonImmutable|null $created_at
+ * @property \Carbon\CarbonImmutable|null $updated_at
+ * @property-read Employee|null $employee
+ *
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task newModelQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task newQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task query()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task whereCreatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task whereDescription($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task whereEmployeeId($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task whereId($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task whereStatus($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task whereTitle($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Task whereUpdatedAt($value)
+ *
+ * @mixin \Eloquent
+ */
+class Task extends Model
+{
+    protected $table = 'tasks';
+
+    protected $guarded = ['id'];
+
```

Comment:
I didn't catch you had an Enum for the Task statuses. In that case, you can do the following

```suggestion
protected function casts(): array
{
    return [
        'status' => TaskStatus::class,
    ]
}
```

This way, your status will always be an Enum instance and you'll know all the possible statuses every time